src/features/eth.js

```diff
@@ -122,12 +115,6 @@ const Eth = () => {
           )}
         </FormProvider>
       </Container>
-      <WrongChainModal
```

Comment:
removing this it will make this issue to happen again:

https://github.com/EveripediaNetwork/iq-bridge-ui/issues/22#issue-907302483

when you click in metamask, you get not message or anything if you are in a different chain.

https://github.com/EveripediaNetwork/iq-bridge-ui/pull/1#issuecomment-851358969